docs/sphinx_docs/Usage/index.md

```diff
@@ -116,10 +121,15 @@ Model generation is disabled by default. There are two recommended ways to enabl
 
     unknown
   ```
-  *Note*: There is no model printed after the second `(check-sat)` since we
+
+  ```{admonition} Note
+
+  There is no model printed after the second `(check-sat)` since we
   don't demand it with the statement `(get-model)`.
+  ```
```

Comment:
Ditto, I think having this one as an admonition is more distracting than anything